src/NzbDrone.Api/NzbDrone.Api.csproj

```diff
@@ -81,11 +81,6 @@
     <Compile Include="..\NzbDrone.Common\Properties\SharedAssemblyInfo.cs">
       <Link>Properties\SharedAssemblyInfo.cs</Link>
     </Compile>
-    <Compile Include="Authentication\AuthenticationService.cs" />
-    <Compile Include="Authentication\EnableAuthInNancy.cs" />
-    <Compile Include="Authentication\AuthenticationModule.cs" />
-    <Compile Include="Authentication\LoginResource.cs" />
-    <Compile Include="Authentication\NzbDroneUser.cs" />
```

Comment:
Authentication/EnableAuthInNancy.cs(42,37): error CS0121: The call is ambiguous between the following methods or properties: `Nancy.AsyncNamedPipelineBase<System.Func<Nancy.NancyContext,System.Threading.CancellationToken,System.Threading.Tasks.Task<Nancy.Response>>,System.Func<Nancy.NancyContext,Nancy.Response>>.AddItemToEndOfPipeline(System.Func<Nancy.NancyContext,Nancy.Response>)' and `Nancy.AsyncNamedPipelineBase<System.Func<Nancy.NancyContext,System.Threading.CancellationToken,System.Threading.Tasks.Task<Nancy.Response>>,System.Func<Nancy.NancyContext,Nancy.Response>>.AddItemToEndOfPipeline(Nancy.PipelineItem<System.Func<Nancy.NancyContext,Nancy.Response>>, bool)'
Authentication/EnableAuthInNancy.cs(43,36): error CS0121: The call is ambiguous between the following methods or properties: `Nancy.AsyncNamedPipelineBase<System.Func<Nancy.NancyContext,System.Threading.CancellationToken,System.Threading.Tasks.Task>,System.Action<Nancy.NancyContext>>.AddItemToEndOfPipeline(System.Action<Nancy.NancyContext>)' and `Nancy.AsyncNamedPipelineBase<System.Func<Nancy.NancyContext,System.Threading.CancellationToken,System.Threading.Tasks.Task>,System.Action<Nancy.NancyContext>>.AddItemToEndOfPipeline(Nancy.PipelineItem<System.Action<Nancy.NancyContext>>, bool)'
Extensions/Pipelines/CorsPipeline.cs(12,36): error CS0121: The call is ambiguous between the following methods or properties: `Nancy.AsyncNamedPipelineBase<System.Func<Nancy.NancyContext,System.Threading.CancellationToken,System.Threading.Tasks.Task>,System.Action<Nancy.NancyContext>>.AddItemToEndOfPipeline(System.Action<Nancy.NancyContext>)' and `Nancy.AsyncNamedPipelineBase<System.Func<Nancy.NancyContext,System.Threading.CancellationToken,System.Threading.Tasks.Task>,System.Action<Nancy.NancyContext>>.AddItemToEndOfPipeline(Nancy.PipelineItem<System.Action<Nancy.NancyContext>>, bool)'
Profiles/Delay/DelayProfileModule.cs(25,50): error CS0411: The type arguments for method `NzbDrone.Api.Validation.RuleBuilderExtensions.EmptyCollection<T,TProp>(this FluentValidation.IRuleBuilder<T,System.Collections.Generic.IEnumerable<TProp>>)' cannot be inferred from the usage. Try specifying the type arguments explicitly
Extensions/Pipelines/CacheHeaderPipeline.cs(18,36): error CS0121: The call is ambiguous between the following methods or properties: `Nancy.AsyncNamedPipelineBase<System.Func<Nancy.NancyContext,System.Threading.CancellationToken,System.Threading.Tasks.Task>,System.Action<Nancy.NancyContext>>.AddItemToStartOfPipeline(System.Action<Nancy.NancyContext>)' and `Nancy.AsyncNamedPipelineBase<System.Func<Nancy.NancyContext,System.Threading.CancellationToken,System.Threading.Tasks.Task>,System.Action<Nancy.NancyContext>>.AddItemToStartOfPipeline(Nancy.PipelineItem<System.Action<Nancy.NancyContext>>, bool)'
Extensions/Pipelines/IfModifiedPipeline.cs(18,37): error CS0121: The call is ambiguous between the following methods or properties: `Nancy.AsyncNamedPipelineBase<System.Func<Nancy.NancyContext,System.Threading.CancellationToken,System.Threading.Tasks.Task<Nancy.Response>>,System.Func<Nancy.NancyContext,Nancy.Response>>.AddItemToStartOfPipeline(System.Func<Nancy.NancyContext,Nancy.Response>)' and `Nancy.AsyncNamedPipelineBase<System.Func<Nancy.NancyContext,System.Threading.CancellationToken,System.Threading.Tasks.Task<Nancy.Response>>,System.Func<Nancy.NancyContext,Nancy.Response>>.AddItemToStartOfPipeline(Nancy.PipelineItem<System.Func<Nancy.NancyContext,Nancy.Response>>, bool)'
Extensions/Pipelines/NzbDroneVersionPipeline.cs(11,36): error CS0121: The call is ambiguous between the following methods or properties: `Nancy.AsyncNamedPipelineBase<System.Func<Nancy.NancyContext,System.Threading.CancellationToken,System.Threading.Tasks.Task>,System.Action<Nancy.NancyContext>>.AddItemToStartOfPipeline(System.Action<Nancy.NancyContext>)' and `Nancy.AsyncNamedPipelineBase<System.Func<Nancy.NancyContext,System.Threading.CancellationToken,System.Threading.Tasks.Task>,System.Action<Nancy.NancyContext>>.AddItemToStartOfPipeline(Nancy.PipelineItem<System.Action<Nancy.NancyContext>>, bool)'